pkgs/development/python-modules/notifications-android-tv/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, httpx
+, poetry-core
+, pytest-asyncio
+, pytest-httpx
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "notifications-android-tv";
+  version = "1.0.0";
+  format = "pyproject";
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "engrbm87";
+    repo = "notifications_android_tv";
+    rev = version;
+    hash = "sha256-Xr+d2uYzgFp/Fb00ymov02+GYnwjGc3FbJ/rIvQXzCE=";
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  checkInputs = [
+    pytest-httpx pytest-asyncio
+  ];
```

Comment:
Please move these below `pythonImportsCheck` and expand list items to one per line.

Reply:
> Please move these below `pythonImportsCheck`

I can do that, of course, but I've noticed that the majority of python modules don't have it in that order:

```
$ pwd
/home/dominikh/prj/nixpkgs/pkgs/development/python-modules

$ rg --count --multiline "[cC]heckInputs(.|\n)*pythonImportsCheck"  | wc -l
2266

$ rg --count --multiline "pythonImportsCheck(.|\n)*[cC]heckInputs"  | wc -l
133
```

Is this a recent change in preferred order?

Reply:
The order in most packages is off. We prefer to have it in execution order and `pythonImportsCheck`  is run before `checkPhase`.

Reply:
Thank you for the explanation, it's appreciated.